_migration-assistant/migration-phases/migrate-metadata/index.md

```diff
@@ -127,7 +96,7 @@ Results:
 ```
 
 
-## Using the migrate command
+## Using the `migrate` command
```

Comment:
Code font uses the same font size no matter if it's in the heading or plain text. Thus, code font appears much smaller in the heading. Therefore, we normally don't use code font in headings. I would remove code font.